extensions/wikia/WikiaMaps/WikiaMaps.i18n.php

```diff
@@ -1335,8 +1301,6 @@
 	'wikia-interactive-maps-create-map-choose-tile-set-tip' => '選擇現有的地圖模板或上傳新的地圖。',
 	'wikia-interactive-maps-create-map-no-tile-set-found' => '很抱歉，沒有找到模板。',
 	'wikia-interactive-maps-create-map-clear-tile-set-search' => '清除搜尋過濾功能',
-	'wikia-interactive-maps-create-map-choose-type-tip' => '選擇實際的地圖(比如:紐約约市)或者自訂地圖(比如:《蝙蝠俠》中的阿甘城)。',
-	'wikia-interactive-maps-create-map-choose-type-tip-link' => '了解更多',
 	'wikia-interactive-maps-create-map-title-label' => '命名地圖',
 	'wikia-interactive-maps-create-map-tile-set-title-label' => '命名模板',
 	'wikia-interactive-maps-create-map-tile-set-title-placeholder' => '命名圖片，方便其他用戶再使用（比如:《蝙蝠俠》中的阿甘城）',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "命名圖片，方便其他用戶再使用（比如:《蝙蝠俠》中的阿甘城）" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)